### ISSUE TITLE
Modifications to live code edit feature persist in memory after code is reset

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-org#1498]
When using the live code sandbox, toggling the code editor removes changes that were made by the user but those modifications still persist in memory. It looks like a generic issue across all the examples.

This causes confusion when experimenting with the live code examples.

Steps to reproduce:

- Expand the first example for [Pagination](https://www.patternfly.org/v4/documentation/react/components/pagination)
-  Change the `perPage` state property to 10
- Toggle the live editor, notice your modification has been reverted back to 20
- Paginate forward through the set, and notice that the previously set value of 10 is being used.

![2019-10-01 18 13 16](https://user-images.githubusercontent.com/5942899/66052680-013bcb00-e4ff-11e9-8ad8-7698a8580ed8.gif)

**Comment by rachael-phillips** [comment]
@redallen is there potential for this to be fixed at all by the site refactor effort?

**Comment by redallen** [comment]
@rachael-phillips Yes it's potentially fixed as there's a new implementation. @seanforyou23 is this still an issue on the new https://patternfly-react-surge.sh/patternfly-4 ?

**Comment by seanforyou23** [comment]
@redallen not sure, that link doesn't seem to work for me

**Comment by redallen** [comment]
@seanforyou23 Fixed the link, sorry. It's just our current docs site.

**Comment by seanforyou23** [comment]
There seems to still be issues with loading that site, can't seem to expand the components section of the nav, and when I can, selecting a menu item doesn't update the main content area. You should be able to simply follow the steps to reproduce mentioned in the issue to see if the problem still exists.

**Comment by seanforyou23** [comment]
This still seems to be a problem. Here's a simpler way to reproduce this issue:

1. Navigate to https://www.patternfly.org/v4/documentation/react/components/notificationbadge#basic
2. Expand the JSX code editor and replace `aria-label="Notifications"` with `aria-label="Foo"`
3. Toggle the JSX code editor button closed and back open again
4. Notice the aria-label is still set to the previous value of "Foo" while the code snippet has been reverted to the original state of "Notifications".

IMO this a real hiccup in the developer experience because we're inviting people to augment the code snippets in real-time but the editor state becomes out of sync with the code very easily and it may not be obvious to a users why the generated example isn't behaving the way it should considering the code snippet.
